_register_database.php

```diff
@@ -37,3 +19,20 @@
 			PHP extension is not available. Please install or enable it and refresh this page.'
     )
 );
+
+// Register MySQL PDO as a database adapter (listed as first option in Dev/Install/config-form.html)
```

Comment:
I _think_ that moving this code down will make the last part of this sentence invalid. Could you remove it?

Reply:
ping @sminnee

Reply:
fixed